src/olympia/versions/tasks.py

```diff
@@ -124,10 +124,18 @@ def extract_version_to_git(
     # we can access them.
     version = Version.unfiltered.get(pk=version_id)
 
+    if (
+        version.addon.type != amo.ADDON_EXTENSION or
+        not version.all_files[0].is_webextension
+    ):
+        log.debug('Skipping git extraction of add-on "{}": not a '
+                  'web-extension.'.format(version.addon.id))
```

Comment:
I don't think this is enough.

You should also exclude those from the management command, so that we never call the extraction function on those versions, otherwise if the first `BATCH_SIZE` versions of an add-on are non-webextensions we'd never extract anything more.

You should still keep this, but maybe make the logging a `warn` instead, cause if the management command is fixed this should be unreachable, so we want to log a little louder if somehow it happens anyway.

Reply:
> You should also exclude those from the management command, so that we never call the extraction function on those versions, otherwise if the first BATCH_SIZE versions of an add-on are non-webextensions we'd never extract anything more.

heh, good catch 😥